tf-modules/ingest/message_template.tf

```diff
@@ -1,3 +1,27 @@
+resource "aws_secretsmanager_secret" "message_template_cmr_password" {
+  name_prefix = "${var.prefix}-message-template-cmr-password"
+  description = "CMR password for the Cumulus message template in the ${var.prefix} deployment"
+  tags        = local.default_tags
+}
+
+resource "aws_secretsmanager_secret_version" "message_template_cmr_password" {
```

Comment:
Hmm, so we're creating 2 different secrets to manage the CMR password: one for the archive module and one for the ingest module? I guess there's no way around that if ultimately we want these modules to be usable independently

Reply:
Correct.